code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -659,8 +666,8 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 /datum/gas_mixture/proc/gas_pressure_quadratic(a, b, c, lower_limit, upper_limit)
 	var/solution
 	if(!IS_INF_OR_NAN(a) && !IS_INF_OR_NAN(b) && !IS_INF_OR_NAN(c))
-		solution = max(SolveQuadratic(a, b, c))
-		if((solution > lower_limit) && (solution < upper_limit)) //SolveQuadratic can return nulls so be careful here
+		solution = max(SolveQuadratic(a, b, c)) 
+		if((solution > lower_limit) && (solution < upper_limit)) //SolveQuadratic can return empty lists so be careful here
```

Comment:
why the ()s here?

Reply:
readability i guess?

Reply:
d4ed90dc9033ec1a732c4042cdc380ffc0b70ac0 should remove those parentheses. I personally like it more the old way, helps me read things quicker while making this, but don't really mind either.